keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -362,41 +357,36 @@ providers, as is the case with the NetworkPolicy API.
-accessible instead because of ClusterNetworkPolicy `Allow` rules.
-
-Developers could face some difficulties figuring out why the NetworkPolicies
-did not take effect, even if they know to look for ClusterNetworkPolicy rules
-that can potentially override these policies:
-To understand why traffic between a pair of Pods is allowed/denied, a list of
-NetworkPolicy resources in both Pods' Namespace used to be sufficient
-(considering no other CRDs in the cluster tries to alter traffic behavior).
-The same Pods, on the other hand, can appear as an AppliedTo, or an ingress/egress
-peer in any ClusterNetworkPolicy. This makes looking up policies that affect a
-particular Pod more challenging than when there's only NetworkPolicy resources.
-
-In case where a ClusterNetworkPolicy `Pass` rule,
-ClusterNetworkPolicy `Deny` rule, ClusterNetworkPolicy `Allow` rule,
+To understand why traffic between a pair of Pods is allowed or denied, a list of 
+NetworkPolicy resources in both Pods' Namespace used to be sufficient (considering 
+no other CRDs in the cluster tries to alter traffic behavior). With the introduction
+of AdminNetworkPolicy, all users could face some difficulties determining why 
+NetworkPolicies did not take effect even if they know to look for, 
+and have the correct permissions to list, any interfering AdminNetworkPolicy rules. 
+
+Also, in the case where a AdminNetworkPolicy `Pass` rule,
+AdminNetworkPolicy `Deny` rule, AdminNetworkPolicy `Allow` rule,
 NetworkPolicy rule and default rule applies to an overlapping
 set of Pods, users will need to refer to the priority associated with the
 rule, to determine which rule would take effect. Figuring out how stacked policies
-affect traffic between workloads might not be very straightfoward.
-
-To mitigate this risk and improve UX, a tool which does a reverse look up and lists rules
-and policies that impact a given Pod and prints out relative precedence of those rules
-can be quite useful. The [cyclonus](https://github.com/mattfenwick/cyclonus)
-project for example, could be extended to support ClusterNetworkPolicy. kubectl plugin extensions
-that allow listing of policies and rules in this manner should be developed. Similarly
-a kubectl plugin extension should be developed that checks whether communication is allowed
-between a specified pair of source and destination pods for traffic of a specified protocol/ port type.
+affect traffic between workloads might not be very straightforward.
+
+To mitigate this risk and improve usability, some additional in-tree tooling
+for both the Admin and Developer will need to be created. For the Admin, it is 
+safe to assume they will have the correct RBAC roles to list all the NetworkPolicies 
+and AdminNetworkPolicies in a cluster.  Therefore, the Admin oriented tooling should 
+be able to both alert the Admin to any overriding of NetworkPolicies that may occur for a given 
+AdminNetworkPolicy and provide a warning if there is another ANP with the same 
```

Comment:
what about "any overriding of NetworkPolicies that may occur if a new AdminNetworkPolicy is to be created"

Reply:
Done